src/components/collapsible-section/collapsible-section.tsx

```diff
@@ -52,6 +63,25 @@ export class CollapsibleSection {
     @Event()
     private action: EventEmitter<Action>;
 
+    @Element()
+    private host: HTMLElement;
+
+    public componentDidRender() {
+        const button = this.host.shadowRoot.querySelector(
+            '.open-close-toggle'
+        ) as HTMLElement;
+
+        makeEnterClickable(button);
```

Comment:
`componentDidRender` is called every single time the component is re-rendered. While `makeEnterClickable` has an early exit for when the supplied element has already been "enhanced", we still query the DOM after every single re-render.

Also, if the implementation of `makeEnterClickable` were ever to change (intentionally or not), so that we no longer exit early but actually attach new listeners each time, even if the same element is passed, we might end up with a pretty awful leak here 😕

So:
1. Does this _really_ have to be called in `componentDidRender`?
2. If it does, please make sure we only do the DOM querying and calling of `makeEnterClickable` _once_ 🙂 (for example by adding a `private initialized = false`, which is checked before in initialising, and flipped to `true` when initialisation is completed)

@LawrenceBorst